docs/mindsdb_sql/agents/chatbot.mdx

```diff
@@ -3,59 +3,73 @@ title: Chatbot
+* `agent` is an [AI agent](/mindsdb_sql/agents/agent) created with the `CREATE AGENT` command. It consists of an AI model trained with defined data sets.
+* `is_running` indicates whether or not to start the chatbot upon creation.
 
-    The parameters include the following:
+<Tip>
+Here are some tips of you use the Slack integration:
+
+1. If you want to use Slack in the [`CREATE CHATBOT`](/agents/chatbot) syntax, use [this method of connecting Slack to MindsDB](/integrations/app-integrations/slack#method-1-chatbot-responds-in-direct-messages-to-a-slack-app).
+
+2. If you want to connect the chatbot to multiple Slack channels, open your Slack application and add the App/Bot to one or more channels:
+  - Go to the channel where you want to use the bot.
+  - Right-click on the channel and select *View Channel Details*.
+  - Select *Integrations*.
+  - Click on *Add an App*.
+</Tip>
+
+## Chatbot with an LLM
 
-    * `database` stores connection to a chat app (like [Slack](/integrations/app-integrations/slack) or [MS Teams](/integrations/app-integrations/microsoft-teams)) that should be created with the `CREATE DATABASE` statement.
-    * `agent` is an [AI agent](/agents/agent) created with the `CREATE AGENT` command. It consists of an AI model trained with defined data sets.
-    * `is_running` indicates whether or not to start the chatbot upon creation.
+ALternatively, you can create a chatbot that is equivalent to embedding an LLM of your choice into a chat app, like [Slack](/integrations/app-integrations/slack) or [MS Teams](/integrations/app-integrations/microsoft-teams).
 
-    <Note>
-    If you want to use Slack in the [`CREATE CHATBOT`](/agents/chatbot) syntax, use [this method of connecting Slack to MindsDB](/integrations/app-integrations/slack#method-1-chatbot-responds-in-direct-messages-to-a-slack-app).
-    </Note>
+Here is how to create a chatbot that integrates an LLM and can be connected to a chat interface to have a conversation with your data.
 
-    <Tip>
-    **How to connect the chatbot to multiple Slack channels?**
+```sql
+CREATE CHATBOT my_chatbot
+USING
+    database = 'my_slack',   -- created with CREATE DATABASE my_slack
+    model = 'my_model',   -- created with CREATE MODEL my_model
+    is_running = true;   -- default is true
+```
 
-    Open your Slack application and add the App/Bot to one or more channels:
-      - Go to the channel where you want to use the bot.
-      - Right-click on the channel and select *View Channel Details*.
-      - Select *Integrations*.
-      - Click on *Add an App*.
-    </Tip>
+The parameters include the following:
 
-* Deleting a chatbot:
+* `database` stores connection to a chat app (like [Slack](/integrations/app-integrations/slack) or [MS Teams](/integrations/app-integrations/microsoft-teams)) that should be created with the `CREATE DATABASE` statement.
+* `model` is a conversational model created with the `CREATE MODEL` command.
```

Comment:
Should we maybe define what a 'conversational model' is and which AI Engines support this? I remember being confused with this in the beginning.

Reply:
Yes. Agents can use only models created via LangChain, so I added this `is a conversational model created with the CREATE MODEL command using the [LangChain engine](/integrations/ai-engines/langchain)`